network/f5/bigip_virtual_server.py

```diff
@@ -0,0 +1,454 @@
+            api.LocalLB.VirtualServer.set_description(virtual_servers =[name],descriptions=[description])
+            updated=True
+        return updated
+    except bigsuds.OperationFailed, e:
+        raise Exception('Error on setting description : %s ' % e)
+
+def get_persistence_profiles(api,name):
+    return api.LocalLB.VirtualServer.get_persistence_profile(virtual_servers = [name])[0]
+
+def set_default_persistence_profiles(api,name,persistence_profile):
+    updated=False
+    if persistence_profile is None:
+        return updated
+    try:
+        current_persistence_profiles = get_persistence_profiles(api,name)
+        default=None
+        for profile in current_persistence_profiles:
+            if profile['default_profile']:
+                default=profile['profile_name']
+                break
+        if default is not None and default != persistence_profile:
+            api.LocalLB.VirtualServer.remove_persistence_profile(virtual_servers=[name],profiles=[[{'profile_name':default,'default_profile' : True}]])
+        if default != persistence_profile:
+            api.LocalLB.VirtualServer.add_persistence_profile(virtual_servers=[name],profiles=[[{'profile_name':persistence_profile,'default_profile' : True}]])
+            updated=True
+        return updated
+    except bigsuds.OperationFailed, e:
+        raise Exception('Error on setting default persistence profile : %s' % e)
+
+def main():
+    argument_spec = f5_argument_spec()
+    argument_spec.update( dict(
+            state = dict(type='str', default='present',
+                         choices=['present', 'absent', 'disabled', 'enabled']),
+            name = dict(type='str', required=True,aliases=['vs']),
+            destination = dict(type='str', aliases=['address', 'ip']),
+            port = dict(type='int'),
+            all_profiles = dict(type='list'),
+            pool=dict(type='str'),
+            description = dict(type='str'),
+            snat=dict(type='str'),
+            default_persistence_profile=dict(type='str')
+        )
+    )
+
+    module = AnsibleModule(
+        argument_spec = argument_spec,
+        supports_check_mode=True
+    )
+
+    (server,user,password,state,partition,validate_certs) = f5_parse_arguments(module)
+    name = fq_name(partition,module.params['name'])
+    destination=module.params['destination']
+    port=module.params['port']
+    all_profiles=fq_list_names(partition,module.params['all_profiles'])
+    pool=fq_name(partition,module.params['pool'])
+    description = module.params['description']
+    snat = module.params['snat']
+    default_persistence_profile=fq_name(partition,module.params['default_persistence_profile'])
+
+    if 1 > port > 65535:
+        module.fail_json(msg="valid ports must be in range 1 - 65535")
+  
+    try:
+        api = bigip_api(server, user, password)
+        result = {'changed': False}  # default
+
+        if state == 'absent':
+                if not module.check_mode:
```

Comment:
Can this block (to line 387) be spaced over to conform with the rest of the code?